aduser/plugins/example/__init__.py

```diff
@@ -30,51 +30,54 @@ def pixel(request):
 
 def update_data(data, request_data):
     # Ignore request_data
-    global schema
+    global taxonomy
 
     input_choices = ['Professor X', 'Deadpool', 'Aquaman', 'professorX']
 
-    data['keywords'].update({'0001': random.choice(schema['values']['0001']['values'])['key']})
-    data['keywords'].update({'0002': random.randint(0, 100)})
-    data['keywords'].update({'0003': random.choice(input_choices)})
-    data['keywords'].update({'0004': bool(random.getrandbits(1))})
-    # data['keywords'] = normalize(data['keywords'])
+    data['keywords'].append({'0001': random.choice(taxonomy['data'][0]['data'])['key']})
+    data['keywords'].append({'0002': random.randint(0, 100)})
+    data['keywords'].append({'0003': random.choice(input_choices)})
+    data['keywords'].append({'0004': bool(random.getrandbits(1))})
 
     data['human_score'] = random.random()
     return data
 
 
-def generate_schema():
-    global schema
-    global schema_name
-    global schema_version
-
-    values = {'0001': {'label': 'Interests',
-                       'type': 'dict',
-                       'values': [{'label': 'Interest: DC', 'key': '0001'},
-                                  {'label': 'Interest: Marvel', 'key': '0002'},
-                                  {'label': 'Interest: Marvel: Spiderman', 'key': '0003'},
-                                  {'label': 'Interest: Marvel: Venom', 'key': '0004'}],
-                       },
-              '0002': {'label': 'Comic books owned',
-                       'type': 'num'},
-              '0003': {'label': 'Favourite superhero',
-                       'type': 'input'},
-              '0004': {'label': 'Registered at comicbookheroes.net',
-                       'type': 'bool'}
-              }
-
-    schema = {'meta': {'name': schema_name,
-                       'ver': schema_version},
-              'values': values}
-#
+def generate_taxonomy():
+    global taxonomy
+    global taxonomy_name
+    global taxonomy_version
+
+    values = [{'label': 'Interests',
+               'key': '0001',
+               'type': 'dict',
+               'data': [{'label': 'Interest: DC', 'key': '0001'},
```

Comment:
This should probably be "value".
```suggestion
               'data': [{'label': 'Interest: DC', 'value': '0001'},
```